src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml

```diff
@@ -47,7 +58,18 @@
                     {
                       <span class="govuk-error-message" asp-validation-for="FamilyName"></span>
                     }
-                    <input class="govuk-input govuk-!-width-two-thirds" id="FamilyName" name="FamilyName" type="text" asp-for="FamilyName" sfa-validation-for="FamilyName" sfa-validationerror-class="govuk-input--error" autocomplete="family-name" spellcheck="false" />
+                    <input
+                        class="govuk-input govuk-!-width-two-thirds"
+                        id="FamilyName"
+                        name="FamilyName"
+                        type="text"
+                        asp-for="FamilyName"
+                        sfa-validation-for="FamilyName"
+                        sfa-validationerror-class="govuk-input--error"
+                        autocomplete="family-name"
+                        spellcheck="false"
+                        maxlength="10000"
```

Comment:
Perhaps this should be the same length as most Family names in the Database e.g. 250